src/md-datatable-row.component.scss

```diff
@@ -1,13 +1,43 @@
+$first-cell-padding: 24px;
+$cell-padding: 56px;
+
 :host {
+  $hover-color: #EEEEEE;
+  $selected-color: #F5F5F5;
```

Comment:
Regarding this change, I have to set the selected/hover colors according to the theme's $accent color I'm sorry. I will add a commit on top of yours to have this set.